packages/payment-experience-api/src/api/createPaymentController.ts

```diff
@@ -39,6 +44,18 @@ export class CreatePaymentController extends AbstractController<
     } = request
 
     const order = await getOrder({ orderId, user })
+
+    let currentDateTime = new Date()
```

Comment:
Creating function to order-backend and using it here to validate makes this easier, and only needed to change code in one place